postman/schemas/paths/simulation-by-id.yaml

```diff
@@ -0,0 +1,64 @@
+# /simulation/{id}
+get:
+    summary: Get all information about a simulation, including the ids of connected components and runs.
+    operationId: getSimulationConfiguration
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+    responses:
+        "200":
+            description: Successfull operation
+            content:
+                application/json:
+                    schema:
+                        $ref: "../schemas/Simulation.yaml#/components/schema/SimulationConfiguration"
+        "401":
+            description: API token missing
+        "404":
+            description: Id not found
+delete:
+    summary: Delete a simulation configuration and every connected component and run.
+    operationId: deleteSimulationConfiguration
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+              type: string
+              format: uuid
+    responses:
+        "204":
+            description: Deleted simulation configuration
+        "401":
+            description: API token missing
+        "404":
+            description: Id not found
+        
+patch:
```

Comment:
Just thought about the following:
What happens when I configure & run a simulation, then alter the config and run simulations again. How would you distinguish the runs before and after the config change?

Reply:
We decided that after executing the simulation, the user is not allowed to edit the configuration. In the future, we'll provide a route to copy a configuration in order to edit an executed one.